test/settings.ini

```diff
@@ -1,4 +1,4 @@
-host=localhost
```

Comment:
Can you try deleting settings.ini? It was used in test_plugin.sh, but since we don't use that script anymore, it should be safe to delete this file.

Reply:
[03c9800](https://github.com/Shard-Labs/starknet-devnet/pull/76/commits/03c9800db3b70b217734b29fab8698aff31f361b)